### PR TITLE
Fix broken searchPlaceholder option

### PR DESCRIPTION
**Pull request by carpii**

### CODE REVIEW

File: src/function/init.js [hunk 1]
```diff
@@ -73,7 +73,7 @@ function($, emojione, blankImg, slice, css_class, emojioneSupportMode, invisible
                         options.search ?
                         function() {
                             self.search = $("<input/>", {
-                                "placeholder": "SEARCH",
+                                "placeholder": options.searchPlaceholder || "",
```
Comment (pederjohnsen) [inline src/function/init.js]: `"placeholder": options.searchPlaceholder` would be enough here no? as options.searchPlaceholder would always be set

Comment (carpii) [reply src/function/init.js]: @pederjohnsen You're right, but I try to code defensively. If someone accidentally passes a null var as the placeholder, at least this way it wont explode :-)

**Comment by joe1chen** [comment]
@mervick Any chance you could publish a new version to jsdelivr with this fix?

**Comment by mervick** [comment]
@joe1chen I don't use jsdelivr at all, if you need you can publish it there

**Comment by mervick** [comment]
released here https://github.com/mervick/emojionearea/releases/tag/v3.2.7

**Pull request merged**